systemvm/patches/debian/config/opt/cloud/bin/cs/CsRedundant.py

```diff
@@ -96,7 +96,8 @@ def _redundant_on(self):
                 d = s.replace(".templ", "")
             CsHelper.copy_if_needed(
                 "%s/%s" % (self.CS_TEMPLATES_DIR, s), "%s/%s" % (self.CS_ROUTER_DIR, d))
-        CsHelper.copy(
```

Comment:
This was recently changed from copy_if_needed to copy in PR 763 / 692 to solve CLOUDSTACK-8725. Are we sure we want to reverse that? Tests are green but there may not be a test for this? Please double check.

Discussion: https://github.com/apache/cloudstack/pull/692
Jira: https://issues.apache.org/jira/browse/CLOUDSTACK-8725
PR: https://github.com/apache/cloudstack/pull/763/files